docs/source/installation.rst

```diff
@@ -136,52 +136,47 @@ Step 3: Install ``torch`` and ``torchvision`` with CPU or GPU Support
 
     .. tabs::
 
-        .. tab:: CPU Support
-
-            If you plan to run ``ivadomed`` on CPU only, install PyTorch per instructions provided below for your specific operating system:
+        .. tab:: PyPI Installation
 
             .. tabs::
 
-                .. tab:: Windows/Linux
-
-                    .. code::
+                .. tab:: Nvidia GPU Support
 
-                       pip install torch==1.8.0+cpu torchvision==0.9.0+cpu --find-links https://download.pytorch.org/whl/torch_stable.html
+                    ``ivadomed`` requires CUDA11 to execute properly. If you have a nvidia GPU, try to look up its Cuda Compute Score `here <https://developer.nvidia.com/cuda-gpus>`_, which needs to be > 3.5 to support CUDA11. Then, make sure to upgrade to nvidia driver to be at least v450+ or newer.
 
-                .. tab:: Mac
+                    If you have a compatible NVIDIA GPU that supports CUDA11 and with the right driver installed try run the following command relevant to your situation:
```

Comment:
Nitpick, but the wording "try run the following command relevant to your situation" feels a bit unnecessary?

I think we could cut it down to just:

```suggestion
                    If you have a compatible NVIDIA GPU that supports CUDA11, and you have a recent enough driver installed, then run the following command:
```

Reply:
Fixed. I am a wordy person. :D